microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md

```diff
@@ -1,436 +1,171 @@
+In Window-to-Visual hosting and Visual hosting, a _visual_ is a basic graphical unit that can be used to compose graphical experiences on Windows. The Windows graphics APIs that expose this functionality and are relevant to WebView2 are `DirectComposition` and `Windows.UI.Composition`. The "visual" in "Visual hosting" can be any one of `IDCompositionVisual`, `IDCompositionTarget`, or `Windows.UI.Composition.Visual`, which are visuals that are exposed through the `DirectComposition` and `Windows.UI.Composition` APIs.  (Window to Visual hosting uses `IDCompositionVisual` specifically.)  See:
+* [Basic concepts](/windows/win32/directcomp/basic-concepts) in the Windows App Development > DirectComposition docs.
+* [Composition visual](/windows/uwp/composition/composition-visual-tree) in the Windows App Development > UWP docs.
 
 
 <!-- ------------------------------ -->
-#### Composition-based rendering
+#### Advantages
 
-For composition-based WebView2 rendering, use the `CoreWebView2Environment` to create a `CoreWebView2CompositionController`. The `CoreWebView2CompositionController` implements all the APIs as `CoreWebView2Controller`, but includes additional APIs specific to composition-based rendering.
+* Different apps that share a WebView2 user data folder can have different DPI awareness.
 
-##### [.NET/C#](#tab/dotnetcsharp)
+* Different apps that share a WebView2 user data folder can have different integrity levels.
 
-* [CoreWebView2CompositionController Class](/dotnet/api/microsoft.web.webview2.core.corewebview2compositioncontroller)
-* [CoreWebView2Environment.CreateCoreWebView2CompositionControllerAsync Method](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.createcorewebview2compositioncontrollerasync)
+* Different apps that share a WebView2 user data folder won't potentially cause each other to hang.
 
-##### [WinRT/C#](#tab/winrtcsharp)
+* When hosting a WebView2 in a VSTO Add-in, changing monitor scale won't potentially cause the app to hang. See [VSTO Add-ins](/visualstudio/vsto/office-solutions-development-overview-vsto#vsto-add-ins) in _Office solutions development overview (VSTO)_.
 
-* [CoreWebView2CompositionController Class](/dotnet/api/microsoft.web.webview2.core.corewebview2compositioncontroller)
-* [CoreWebView2Environment.CreateCoreWebView2CompositionControllerAsync Method](/dotnet/api/microsoft.web.webview2.core.corewebview2environment.createcorewebview2compositioncontrollerasync)
 
-##### [Win32/C++](#tab/win32cpp)
+<!-- ------------------------------ -->
+#### Disadvantages
 
-* [ICoreWebView2CompositionController interface](/microsoft-edge/webview2/reference/win32/icorewebview2compositioncontroller)
-* [ICoreWebView2Environment::CreateCoreWebView2CompositionController method](/microsoft-edge/webview2/reference/win32/icorewebview2environment3#createcorewebview2compositioncontroller)
-
----
+Enabling Window to Visual hosting mode removes support for Windows Shell Handwriting within the WebView2.
```

Comment:
Windows Shell Handwriting: link to these webpages?
https://learn.microsoft.com/windows/win32/input_ink/input-ink-portal
https://learn.microsoft.com/windows/win32/api/shellhandwriting/
 
Search for articles at LMC:
https://learn.microsoft.com/en-us/search/?terms=Shell%20Handwriting

Reply:
I linked to the two handwriting.h/Ink articles.

rendered section for review:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/windowed-vs-visual-hosting?branch=pr-en-us-3271#disadvantages-1
@johna-ms